content/docs/hooks-effect.md

```diff
@@ -133,21 +133,21 @@ function Example() {
   });
 ```
 
-We declare the `count` state variable, and then we tell React we need to use an effect. We pass a function to the `useEffect` Hook. This function we pass *is* our effect. Inside our effect, we set the document title using the `document.title` browser API. We can read the latest `count` inside the effect because it's in the scope of our function. When React renders our component, it will remember the effect we used, and then run our effect after updating the DOM. This happens for every render, including the first one.
+Kita mendeklarasikan *`count` state variable*, lalu kita memberitahu pada *React* kita perlu menggunakan *effect*. Kita memberikan *function* kepada `useEffect` *Hook*. *Function* ini kita berikan *kepada* *effect* kita. didalam *effect* kita, kita mengatur judul dokumen menggunakan `document.title` browser API. Kita dapat membaca  `count` terakhir di dalam *effect* karena berada dalam lingkup fungsi kita. Ketika *React renders* komponen kita, itu akan mengingatkan *effect*  yang kita gunakan, lalu menjalankan *effect* setelah memperbarui DOM. Ini terjadi untuk setiap *render*, termasuk yang pertama di *render*.
 
-Experienced JavaScript developers might notice that the function passed to `useEffect` is going to be different on every render. This is intentional. In fact, this is what lets us read the `count` value from inside the effect without worrying about it getting stale. Every time we re-render, we schedule a _different_ effect, replacing the previous one. In a way, this makes the effects behave more like a part of the render result -- each effect "belongs" to a particular render. We will see more clearly why this is useful [later on this page](#explanation-why-effects-run-on-each-update).
+Pengembang JavaScript yang berpengalaman mungkin memperhatikan bahwa *function* yang kita berikan kepada `useEffect` akan berbeda setiap render. Ini disengajai. Faktanya, inilah yang memungkinkan kita membaca nilai `count` dari dalam *effect* tanpa mengkhawatikan tentang getting stale. Setiap kali kita mengulangi-*render*, kita menjadwalkan *effect* yang berbeda, menggantikan yang sebelumnya. Di satu sisi, inilah yang membuat *effects* berjalan lebih dari bagian hasil render -- setiap *effect* memiliki render tertentu. Kita akan melihat lebih jelas mengapa itu sangat berguna [nanti pada halaman ini](#explanation-why-effects-run-on-each-update).
```

Comment:
```suggestion
Pengembang JavaScript yang berpengalaman mungkin memperhatikan bahwa *function* yang kita berikan kepada `useEffect` akan berbeda setiap *render*. Ini disengajai. Faktanya, inilah yang memungkinkan kita membaca nilai `count` dari dalam *effect* tanpa mengkhawatikan nya menjadi *stale*. Setiap kali kita me-*render* ulang, kita menjadwalkan *effect* yang berbeda, menggantikan yang sebelumnya. Di satu sisi, inilah yang membuat *effects* berjalan lebih dari bagian hasil render -- setiap *effect* memiliki render tertentu. Kita akan melihat lebih jelas mengapa itu sangat berguna [nanti pada halaman ini](#explanation-why-effects-run-on-each-update).
```

* re-render -> me-*render* ulang

Also need to find a proper translation for "stale". @grikomsn